src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -102,7 +102,7 @@ function createPaymentRequest(args) {
       }
     }
     // Add empty browserInfo for GooglePay
-    if (paymentMethodType === 'paywithgoogle') {
+    if (paymentMethodType === 'paywithgoogle' || paymentMethodType === 'googlepay') {
       paymentRequest.browserInfo = {};
```

Comment:
Is the browserInfo not yet included in the `state.data`?

Reply:
you're right it was included in v4.2.0 https://github.com/Adyen/adyen-web/pull/847